### DISCUSSION TITLE
Will there be a vision pro version of stellarium in the future?

### DISCUSSION THREAD

**Discussion by GitHub-LLL** [Stellarium/stellarium#3889] [Q&A] [answered]
stellarium combine VR may have better experience

**Comment by gzotti** [comment, marked as answer]
Oh really? Then why not show your version?

**Reply by GitHub-LLL** [reply]
sure！I plan to do so, and I hope to become one of the many developers soon.^_^

**Reply by gzotti** [reply]
Note the existing Oculus plugin. It's a start, but probably already based on outdated support libraries.
